index.bs

```diff
@@ -318,6 +318,23 @@ but in some cases the feature should also be detectable
 in the language where it is used
 (such as ''@supports'' in CSS).
 
+<h3 id="restriction-options">Use appropriate methods for restricting features</h3>
+
+The Web has a number of somewhat-related features:
+
+* The <{iframe}> element's <{iframe/sandbox}> attribute
+* [Feature policy](https://w3c.github.io/webappsec-feature-policy/) (and the the <{iframe}> element's <{iframe/allow}> attribute)
+* The <{iframe}> element's <{iframe/allowfullscreen}> and <{iframe/allowpaymentrequest}> attributes
+* The [{{SecureContext}}] WebIDL attribute.
+* [Content Security Policy (CSP)](https://w3c.github.io/webappsec-csp/)
+* [Referrer Policy](https://w3c.github.io/webappsec-referrer-policy/)
+* [Document Policy](https://w3c.github.io/webappsec-feature-policy/document-policy.html)
+
+TODO: Separate out the ones that are not yet interoperable into their own category.
```

Comment:
If it's not interoperable yet, probably best to leave out of the principles document. We also need to write down some guidance here, rough (unpolished) ideas sketched out here: https://github.com/w3ctag/design-principles/issues/41#issuecomment-1074950175

Reply:
Instead of having a complete list here, there should be a list of example (marked as such) like the existing list, then link to another document containing a maintained list